examples/find_nodes.rs

```diff
@@ -196,6 +196,10 @@ async fn main() {
                     Discv5Event::SessionEstablished(enr, _) => info!("Session established {}", enr),
                     Discv5Event::SocketUpdated(addr) => info!("Socket updated {}", addr),
                     Discv5Event::TalkRequest(_) => info!("Talk request received"),
+                    Discv5Event::NATUpdated(addr) => info!("Node is behind NAT, externally reachable address updated {}", addr),
+                    Discv5Event::NATSymmetricUpdated(ip) => info!("Node is behind a symmetric NAT, externally reachable ip updated {}", ip),
```

Comment:
Yes that's right. But it is useful for us to keep the peer in our kbuckets, so that we can send requests to it. Then we can keep the rule "only ENRs with a contactable address in kbuckets" and that way we can also limit the number of symmetrically nat:ed peers based on the ENR.